cloudinit/settings.py

```diff
@@ -41,6 +41,7 @@
         "Bigstep",
         "Scaleway",
         "Hetzner",
+        "NWCS",
```

Comment:
Let's put this at the bottom of the list before "None". We generally try to add new datasources to the end of our datasource detection so that it can't interfere with a previously working datasource detection.

You'll also need to update the datasource list at `tests/unittests/sources/test_common.py` accordingly or we'll get a test failure.

Reply:
Should be resolved in latest push